File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> result;
        int m=*max_element(candies.begin(), candies.end());
        for(int i=0;i<candies.size();i++){
            int s=candies[i]+ extraCandies;
            if(s>=m){
                result.push_back(true);
            }
            else{
                result.push_back(false);
            }
        }
        return result;
    }
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        ans = []
        max_candy = max(candies)
        for i in range(len(candies)):
            curr_sum = candies[i] + extraCandies
            if curr_sum >= max_candy:
                ans.append(True)
            else:
                ans.append(False)
        return ans